cranelift/filetests/filetests/runtests/simple.clif

```diff
@@ -0,0 +1,18 @@
+test run-zkasm
+
+
+function %add_i64(i64, i64) -> i64 {
+block0(v0: i64,v1: i64):
+    v2 = iadd v0, v1
+    return v2
+}
+; run: %add_i64(0, 0) == 0
+; run: %add_i64(0, 1) == 1
+
+function %sub_i64(i64, i64) -> i64 {
+block0(v0: i64,v1: i64):
+    v2 = isub v0, v1
+    return v2
+}
+; run: %add_i64(0, 0) == 0
+; run: %add_i64(0, 1) == -1
```

Comment:
Shouldn't these call `%sub_i64`?

Reply:
Yes, thanks (= But this file is not supposed to really test something, it is more for me to debug the filetest